html/js/guiutils/Tools.js

```diff
@@ -0,0 +1,46 @@
+GAMBIT.UI = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new Tools object.
+    * @class
+    */
+    function Tools() {
+    }
+
+    /**
+    * Function called when New button is pressed.
+    * It creates a new Tree and draws it
+    */
+    Tools.prototype.newTree = function() {
+        var root = new GAMBIT.TREE.Node(null);
+        GAMBIT.tree = new GAMBIT.TREE.Tree(root);
+        GAMBIT.tree.updatePositions();
+        // Create a node and draw it
+        GAMBIT.tree.draw();
+    };
+
+    /**
+    * Function that swithes mode to the one specified by the button pressed
+    * @param {Button} button Button pressed that will activate mode
+    */
+    Tools.prototype.switchMode = function(button){
```

Comment:
I would pass the mode as a parameter instead of the button, that way in the future you can call this function from anywhere.